src/list.coffee

```diff
@@ -56,6 +57,11 @@ class List extends Command
       tree packages, (pack) =>
         packageLine = pack.name
         packageLine += "@#{pack.version}" if pack.version?
+        if pack.apmInstallSource?.type is 'git'
+          repo = getRepository(pack)
+          shaLine = "##{pack.apmInstallSource.sha.substr(0, 8)}"
```

Comment:
This is a silly nit, but to me the hash prefix indicates an issue number or hex color or some such. Using it for a SHA looks weird to my eyes.